docs/execution-providers/QNN-ExecutionProvider.md

```diff
@@ -43,7 +43,7 @@ Note: Starting version 1.18.0 , you do not need to separately download and insta
     - Python 3.11.x
     - Numpy 1.25.2 or >= 1.26.4
   - Install: `pip install onnxruntime-qnn`
-  - Install nightly package `python -m pip install -i https://aiinfra.pkgs.visualstudio.com/PublicPackages/_packaging/ORT-Nightly/pypi/simple/ort-nightly-qnn`
+  - Install nightly package `python -m pip install -i https://aiinfra.pkgs.visualstudio.com/PublicPackages/_packaging/ORT-Nightly/pypi/simple/onnxruntime-qnn`
```

Comment:
Add --pre

Reply:
Missing package name. Also please double check this nightly feed, it is not working in my test:

(py311) >python -m pip install --pre onnxruntime-qnn -i https://aiinfra.pkgs.visualstudio.com/PublicPackages/_packaging/ORT-Nightly/pypi/simple/onnxruntime-qnn
Looking in indexes: https://aiinfra.pkgs.visualstudio.com/PublicPackages/_packaging/ORT-Nightly/pypi/simple/onnxruntime-qnn
ERROR: Could not find a version that satisfies the requirement onnxruntime-qnn (from versions: none)
ERROR: No matching distribution found for onnxruntime-qnn

Reply:
index url should be https://aiinfra.pkgs.visualstudio.com/PublicPackages/_packaging/ORT-Nightly/pypi/simple/ 
e.g.
```
python -m pip install --pre onnxruntime-qnn -i https://aiinfra.pkgs.visualstudio.com/PublicPackages/_packaging/ORT-Nightly/pypi/simple/ 
```

Reply:
Fixed